src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length-29 calendar--start-day-sun">
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), BEM block modifiers should be formatted without the block name when used with the block itself, e.g., `calendar--29` instead of `calendar--month-length-29`.